frontend/src/components/InfoTooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import classNames from 'classnames';
+import css from './InfoTooltip.module.scss';
+
+const classes = classNames.bind(css);
+
+export const InfoTooltip: React.FC<{
+  title: string;
+  children?: any;
+}> = ({ title, children }) => {
+  return (
+    <Tooltip
+      classes={{
+        arrow: classNames(css.tooltipArrow),
+        tooltip: classNames(css.tooltip),
```

Comment:
Also noticed, this should probably use `classes`
```suggestion
        arrow: classes(css.tooltipArrow),
        tooltip: classes(css.tooltip),
```